paseos/power/is_in_eclipse.py

```diff
@@ -21,7 +21,7 @@ def is_in_eclipse(
     Returns:
         bool: True if actor is in eclipse
     """
-    logger.debug(f"Checking whether {actor} is in eclipse at {t}.")
+    logger.debug(f"Checking whether {actor} is in eclipse at {t.mjd2000/pk.SEC2DAY}.")
```

Comment:
Please don't change this. I don't know if it is day at 1482579s (old code shows date and time)

Reply:
Please revert 🙏

Reply:
Still open